CMakeLists.txt

```diff
@@ -260,6 +272,8 @@ endif()
 set(CMAKE_CXX_STANDARD ${QMC_CXX_STANDARD})
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CUDA_STANDARD ${QMC_CUDA_STANDARD})
```

Comment:
Let us use the same QMC_CXX_STANDARD  for CUDA as well.

Reply:
I removed CMAKE_CUDA_STANDARD and replaced it with QMC_CXX_STANDARD